src/poly/poly.jl

```diff
@@ -1,13 +1,13 @@
 export spoly, PolyRing, change_base_ring, coeff, coeffs, coeffs_expos,
        content, deflation, deflate, degree, degrees, degree_bound,
        derivative, div, divides, direm, evaluate, exponent, exponent!,
-       exponent_vectors, finish, gen, has_global_ordering,
+       exponent_vectors, factor, finish, gen, has_global_ordering,
        inflate, isgen,
        ismonomial, isterm, jacobi, jet, lc, lt, lm, lead_exponent,
        monomials, MPolyBuildCtx,
        nvars, ordering, @PolynomialRing, primpart,
-       push_term!, remove, sort_terms!, symbols, terms,
-       total_degree, valuation, var_index, vars
+       push_term!, remove, sort_terms!, squarefree_factorization,
```

Comment:
Can I suggest the name factor_squarefree. This seems to agree with the convention the Hecke people are using for various factoring routines. It also gets around needing to decide on British vs Americano spelling.

Reply:
name is changed.